webknossos-datastore/app/com/scalableminds/webknossos/datastore/dataformats/zarr/ZarrCoordinatesParser.scala

```diff
@@ -26,12 +28,18 @@ object ZarrCoordinatesParser {
       (x, y, z) = (parsedCoordinates(parsedCoordinates.length - 3),
                    parsedCoordinates(parsedCoordinates.length - 2),
                    parsedCoordinates(parsedCoordinates.length - 1))
-      additionalCoordinates = if (parsedCoordinates.length > 4)
+      reorderedAdditionalAxes = reorderedAdditionalAxesOpt.getOrElse(List.empty)
+      _ <- bool2Fox(reorderedAdditionalAxes.length == parsedCoordinates.length - 4) ?~> "zarr.invalidAdditionalCoordinates" ~> NOT_FOUND
+      sentAdditionalCoordinates = parsedCoordinates.length > 4
```

Comment:
```suggestion
      requestContainsAdditionalCoordinates = parsedCoordinates.length > 4
```
`sent` is not immediately recognizable as a boolean. I always like bools to be obvious in the variable naming :)